GVFS/GVFS.Common/Tracing/TelemetryDaemonEventListener.cs

```diff
@@ -118,7 +110,7 @@ private void SendMessage(string message)
                 catch (TimeoutException)
                 {
                     // We can't connect; we will try again with a new pipe on the next message
-                    return;
+                    throw;

```

Comment:
We have been using the `bool Try...` pattern but I think that can be propagated too much and make the code a lot less readable.  If we decide to keep the exception pattern here, I would just remove the try/catch here altogether.

Reply:
Will remove the `try..catch` here.